scipy/signal/_signaltools.py

```diff
@@ -4108,11 +4108,11 @@ def filtfilt(b, a, x, axis=-1, padtype='odd', padlen=None, method='pad',
     argument.  The difference between `y1` and `y2` is small.  For long
     signals, using `irlen` gives a significant performance improvement.
 
-    >>> x = rng.standard_normal(5000)
+    >>> x = rng.standard_normal(4000)
```

Comment:
What was the warning? Anything to worry about?

Reply:
I think it's ok it was: `numpy.linalg.LinAlgError: SVD did not converge in Linear Least Squares`

Reply:
I don't think we need to worry about this. It's just that for *some* random seeds, the calculation is less stable. See: https://github.com/numpy/numpy/issues/22664